Environment/Environment.tex

```diff
@@ -419,18 +423,18 @@ \subsubsection{Method overridings}
 		{\includegraphics[scale=0.7]{inheritanceOverriding}}
 	\end{center}
 	\caption{\ct{ScaleMorph>>>defaultColor} and the methods that it overrides, in inheritance order.
-	The \lct{inheritance} button is gold because the displayed method is overridden in a subclass.}
+	The \lct{inheritance} button is pink because the displayed method is overridden in a subclass.}
 	\label{fig:inheritanceOverriding}
 \end{figure}
 
 
 \subsubsection{The hierarchy browser}
 \label{sec:hierarchy}
 
-The \button{hierarchy} button opens a \ind{hierarchy browser} on the current class; this browser can also be opened by using the \menu{browse hierarchy} menu item in the class pane.
-The hierarchy browser is similar to the system browser, but instead of displaying the system categories and the classes in each category, it shows a single list of classes, indented to represent inheritance.
-The category of the selected class is displayed in the small annotation pane at the top of the browser.
-The hierarchy browser is designed to make it easy to navigate through the inheritance hierarchy, but does not show all of the classes in the system: only the superclasses and subclasses of the initial class are shown.
+The \button{hierarchy} button opens a \ind{hierarchy browser} on the current class; this browser can also be opened by using the \menu{browse hierarchy} menu item in the yellow button menu of the class pane.
+The hierarchy browser is similar to the system browser, but instead of displaying the system categories and the classes in each category, it shows a single list of classes indented to represent inheritance.
+The category of the selected class is displayed in the title of the browser window.

```

Comment:
@codeZeilen Wait, where is that? Afaik this only works for *environments* (see Browser>>#defaultBrowserTitle).

![image](https://github.com/hpi-swa-lab/SqueakByExample-english/assets/38782922/dff3a80d-08c2-4665-b33d-e6251c2a2a21)